cmd/geth/main.go

```diff
@@ -379,46 +386,41 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 
 // Starts the permission services. services will come up only when
 // geth is brought up in --permissioned mode and permission-config.json is present
-func startQuorumPermissionService(ctx *cli.Context, stack *node.Node) {
+func startQuorumPermissionService(ctx *cli.Context, stack *node.Node) error {
 
 	var quorumApis []string
 	dataDir := ctx.GlobalString(utils.DataDirFlag.Name)
-	permEnabled := ctx.GlobalBool(utils.EnableNodePermissionFlag.Name)
 
 	var permissionConfig types.PermissionConfig
 	var err error
-	if permEnabled {
-		if permissionConfig, err = permission.ParsePermissionConifg(dataDir); err != nil {
-			log.Error("loading of permission-config.json failed", "error", err)
-			return
-		}
-	} else {
-		// permissions not enabled hence none of the services will be available.
-		return
+
+	if permissionConfig, err = permission.ParsePermissionConifg(dataDir); err != nil {
+		log.Error("loading of permission-config.json failed", "error", err)
+		return nil
```

Comment:
```suggestion
return err
```

Reply:
If permissions-config.json is not present, this means that new permissions model is not implemented and hence will continue with the earlier checks of permissioned-nodes.json. Hence returning nil logging the error. If error is returned it will bring down the node.